### DISCUSSION TITLE
zstandard error plus error saying water already taken

### DISCUSSION THREAD

**Discussion by JahanzebTariq** [PMEAL/OpenPNM#2677] [Q&A]
Greetings professor @jgostick and @ma-sadeghi .. I currently ran into this error which I didn't encounter before. I ran the code 10 times but never got this error but I've been getting this error since yesterday... please address this issue

water = op.phase.Water(network=pn, name='water')
water.add_model_collection(op.models.collections.phase.water)
water.add_model_collection(op.models.collections.physics.basic)
water.regenerate_models()

![image](https://user-images.githubusercontent.com/99289270/215243389-544c2e88-7a82-4006-b3b2-bf959e255651.png)
![image](https://user-images.githubusercontent.com/99289270/215243470-0b7801fe-a85e-4b1b-aaab-f1a3123dfff6.png)

please help me with this issue

**Comment by jgostick** [comment]
Usually when something like this happens, it's because you installed some other package, possibly unrelated to OpenPNM, and it updated/upgraded some dependency that it needed, which broke something else.  The only real solution is to create a new virtual environment and reinstall openpnm.
